### ISSUE TITLE
No visible indication of keyboard focus - accessibility issue

### ISSUE THREAD

**Issue by brichwin** [tinymce/tinymce-docs#1538]
**Documentation/Product Version**
_TinyMCE 4 Documentation_ OR _TinyMCE 5 Documentation_

**Links to Page(s) affected**
https://www.tiny.cloud/docs/advanced/keyboard-shortcuts/
All pages, really

**Description of Issue**
Ironically, the keyboard-shortcuts page, the accessibility page, and all pages really are not accessible to keyboard only users because no visible indication of elements having keyboard focus is provided. Sighted keyboard only users cannot tab to navigate the page's controls because they cannot tell which control they are on. For instance, tabbing down the left-hand side navigation links. 

The applicable WCAG 2.1 Success Criteria is: 2.4.7 Focus Visible: Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible. (Level AA)

**Suggested Fix**
When a user interface control receives focus, a visible border should be displayed around it. This is likely solved by modifying the site's CSS.

See: [Focus Visible: Understanding SC 2.4.7](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html) for more information.

**Comment by tylerkelly13** [comment]
Thanks @brichwin !

I've passed this issue on to our website maintainers to see what we can do. 

Cheers
Tyler

**Comment by tylerkelly13** [comment]
Hi @brichwin,

Sorry for the slow response. We've released a few changes to the docs site to add the keyboard focus, as well as improve keyboard navigation of the navigation pane on the left hand side of the site.

Thanks
Tyler
